src/test/unit_tests/test_bucketspace_fs.js

```diff
@@ -954,6 +955,65 @@ mocha.describe('bucketspace_fs', function() {
             assert.deepEqual(output_lifecycle, []);
         });
     });
+
+    mocha.describe('bucket policy ownership validations', function() {
+        mocha.before(async function() {
+            const param = { name: test_bucket3 };
+            await bucketspace_fs.create_bucket(param, dummy_object_sdk);
+        });
+        mocha.after(async function() {
+            await fs_utils.folder_delete(`${new_buckets_path}/${test_bucket}`);
+            const file_path = get_config_file_path(CONFIG_SUBDIRS.BUCKETS, test_bucket3);
+            await fs_utils.file_delete(file_path);
+        });
```

Comment:
Could you please a new line here? (after the `mocha.after` and before the test `mocha.it`)

Reply:
lint error